pkg/build/builder/sti.go

```diff
@@ -33,7 +33,7 @@ func NewSTIBuilder(client DockerClient, dockerSocket string, authCfg docker.Auth
 func (s *STIBuilder) Build() error {
 	tag := s.build.Parameters.Output.DockerImageReference
 	request := &stiapi.Request{
-		BaseImage:    s.build.Parameters.Strategy.STIStrategy.Image,
+		BaseImage:    s.build.Parameters.Strategy.STIStrategy.From.Name,
```

Comment:
So i think you're doing the right thing below (setting the Image field), but you should be using it up here.